index.html

```diff
@@ -1,361 +1,438 @@
+            <textarea
+              id="textarea"
+              class="message"
+              name="message"
+              maxlength="500"
+              placeholder="Enter text here"
+              required
+            ></textarea>
+          </li>
+          <li>
+            <button class="submit-button" type="submit">Get in touch</button>
+          </li>
         </ul>
-    </form>
-    <form class="form2" action="https://formspree.io/f/xvoljkdk" method="post">
+      </form>
+      <form
+        class="form2"
+        action="https://formspree.io/f/xvoljkdk"
+        method="post"
+      >
         <ul class="form">
-            <li class="one">
-                <input class="full-name " type="text" name="first-name" maxlength="30" placeholder="First name" required>
-            </li>
-            <li class="two">
-                <input class="full-name two" type="text" name="last-name" maxlength="30" placeholder="Last name" required>
-            </li>
-            <li class="three">
-                <input class="full-name " type="email" name="email" placeholder="Email adderss" required>
-            </li>
-            <li class="four">
-                <textarea class="message "  name="message" maxlength="500"  required> Hello Mariane. Glad to see you |
-                </textarea>
-            </li>
-            <li class="five">
-                <button class="submit-button "   type="submit">Get in touch</button>
-            </li>
+          <li class="one">
+            <label class="hidden-label" for="fname">First Name</label>
+            <input
+              id="fname"
+              class="full-name"
+              type="text"
+              name="first-name"
+              maxlength="30"
+              placeholder="First name"
+              required
+            />
+          </li>
+          <li class="two">
+            <label class="hidden-label" for="lname">last Name</label>
+            <input
+              id="lname"
+              class="full-name two"
+              type="text"
+              name="last-name"
+              maxlength="30"
+              placeholder="Last name"
+              required
+            />
+          </li>
+          <li class="three">
+            <label class="hidden-label" for="Email-Address"
+              >Email Address</label
+            >
+            <input
+              id="Email-Address"
+              class="full-name"
+              type="email"
+              name="email"
+              placeholder="Email adderss"
+              required
+            />
+          </li>
+          <li class="four">
+            <label class="hidden-label" for="text-area">Enter Text Here</label>
+            <textarea
+              class="message"
+              id="text-area"
+              name="message"
+              maxlength="500"
+              required
+            >
+ Hello Mariane. Glad to see you |
+                </textarea
+            >
```

Comment:
- [ **OPTIONAL** ] **RECOMENDED:** I think it would be better if you use the ` Hello Mariane. Glad to see...` in a `placeholder="..."` attribute and delete it where it is now as it requires the user to every time delete the test itself before he write his own message. I suggest you change it to  `placeholder=" Enter Text Here"` as the current one(`Hello Mariane. Glad to...`) was an example of a message from your template.